common/djangoapps/student/admin.py

```diff
@@ -53,7 +53,16 @@
 # This switch exists because the CourseEnrollment admin views make DB queries that impact performance.
 # In a large enough deployment of Open edX, this is enough to cause a site outage.
 # See https://openedx.atlassian.net/browse/OPS-2943
-COURSE_ENROLLMENT_ADMIN_SWITCH = LegacyWaffleSwitch(STUDENT_WAFFLE_NAMESPACE, 'courseenrollment_admin', __name__)
+# .. toggle_name: student.courseenrollment_admin
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: This toggle will enable the rendering of the admin view of the CourseEnrollment model..
+# .. toggle_warnings: Enabling this toggle may impact in the performance of the platform.
+#     The CourseEnrollment admin view make DB queries that could cause site outages for a large enough
+#     openedx installation.
+# .. toggle_use_cases: opt_in, open_edx
+# .. toggle_tickets: https://openedx.atlassian.net/browse/OPS-2943
```

Comment:
@robrap  I don't have access to this link. Maybe it would be better to use https://github.com/edx/edx-platform/pull/18638 right?